frontend/components/FieldImage/FieldImageEdit.jsx

```diff
@@ -106,20 +106,25 @@ export default class FieldImageEdit extends Component {
       value
     } = this.props
     const displayName = schema.label || schema._id
-    const src = this.getImageSrc(value)
-    const isReference = schema.type === 'Reference'
     const fieldLocalType = schema.publish && schema.publish.subType ? schema.publish.subType : schema.type
     const href = buildUrl(...onBuildBaseUrl(), 'select', schema._id)
+    const isReference = schema.type === 'Reference'
+    const singleFile = schema.settings && schema.settings.limit === 1
+    const values = value && !(value instanceof Array) ? [value] : value
```

Comment:
Sure. I think we're using `instanceof` in a few places, will need to change those too.